v7: `index.d.ts` contains import from nonexistent files

### Describe the bug

Hi, when updating to v7, I get this error:

```
node_modules/vite-plugin-istanbul/dist/index.d.ts(1,34): error TS2307: Cannot find module '/home/runner/work/vite-plugin-istanbul/vite-plugin-istanbul/node_modules/.pnpm/@types+babel__generator@7.6.8/node_modules/@types/babel__generator/index.d.ts' or its corresponding type declarations.
```

Looking at `dist/index.d.ts`, I see 

```ts
import { GeneratorOptions } from '/home/runner/work/vite-plugin-istanbul/vite-plugin-istanbul/node_modules/.pnpm/@types+babel__generator@7.6.8/node_modules/@types/babel__generator/index.d.ts';
```
as the first line

### Reproduction

https://github.com/marekdedic/imagelightbox/pull/1240

### Steps to reproduce

`npm ci && npm run lint`

### System Info

```shell
System:
    OS: Linux 6.8 Ubuntu 24.04.2 LTS 24.04.2 LTS (Noble Numbat)
    CPU: (12) x64 12th Gen Intel(R) Core(TM) i7-1265U
    Memory: 7.02 GB / 15.23 GB
    Container: Yes
    Shell: 5.9 - /usr/bin/zsh
  Binaries:
    Node: 22.11.0 - /usr/bin/node
    Yarn: 1.22.21 - /usr/bin/yarn
    npm: 11.1.0 - /usr/bin/npm
    pnpm: 8.15.8 - /usr/bin/pnpm
  npmPackages:
    vite: ^6.0.2 => 6.1.0
    vite-plugin-istanbul: ^7.0.0 => 7.0.0
```

### Used Package Manager

npm

### Logs

_No response_

### Validations

- [x] Read the [Contributing Guidelines](https://github.com/iFaxity/vite-plugin-istanbul/blob/main/CONTRIBUTING.md).
- [x] Check that there isn't [already an issue](https://github.com/iFaxity/vite-plugin-istanbul/issues) that reports the same bug to avoid creating a duplicate.
- [x] Make sure this is a vite-plugin-istanbul issue and not a framework-specific issue. For example, if it's a Vue SFC related bug, it should likely be reported to [vuejs/core](https://github.com/vuejs/core) instead.
- [x] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

--- a/package.json
+++ b/package.json
@@ -42,6 +42,7 @@
   "dependencies": {
     "@babel/generator": "^7.28.0",
     "@istanbuljs/load-nyc-config": "^1.1.0",
+    "@types/babel__generator": "7.6.8",
     "espree": "^10.3.0",
     "istanbul-lib-instrument": "^6.0.3",
     "picocolors": "^1.1.1",
@@ -57,7 +58,6 @@
     "@ianvs/prettier-plugin-sort-imports": "4.4.1",
     "@semantic-release/changelog": "6.0.3",
     "@semantic-release/git": "10.0.1",
-    "@types/babel__generator": "7.6.8",
     "@types/node": "22.10.7",
     "@types/ws": "8.5.13",
     "husky": "9.1.7",

